server/lib/imageproxy.ts

```diff
@@ -160,6 +160,10 @@ class ImageProxy {
     this.headers = options.headers || null;
   }
 
+  public getHeaders(): HeadersInit | null {
+    return this.headers;
+  }
+
```

Comment:
What's the point of this function?

Reply:
to get the headers?

Reply:
Yeah but I don't see it used anywhere

Reply:
Oh yeah lol. I forgot I already refactored it...